core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -172,8 +172,6 @@ var TopicsAndSkillsDashboardPage = function() {
       'Assign skill to topic button', assignSkillToTopicButtons.first());
 
     var topic = element(by.cssContainingText(topicNamesClass, topicName));
-    await waitFor.elementToBeClickable(
-      topic, 'Topic list item taking too long to be clickable');
     await action.click('Topic list item', topic);
     await action.click('Confirm move button', confirmMoveButton);
     await waitFor.invisibilityOf(
```

Comment:
Ditto (waitFor before .count()) below for:

* `expect(await topicEditOptions.count()).toEqual(1);`
* `expect(await skillEditOptions.count()).toEqual(1);`